app/api/views/dashboards.py

```diff
@@ -50,3 +50,136 @@ def seller_dashboard():
     supplier_responses = briefs.get_supplier_responses(current_user.supplier_code)
 
     return jsonify(items=supplier_responses, supplier={'name': supplier.name, 'code': supplier.code}), 200
+
+
+@api.route('/buyers/my-briefs', methods=['GET'])
+@login_required
+@role_required('buyer')
+def get_buyer_dashboard_data_for_my_briefs():
+    """Buyer dashboard (My briefs) (role=buyer)
+    ---
+    tags:
+      - dashboard
+    definitions:
+      BuyerDashboardItems:
+        type: object
+        properties:
+            items:
+              type: array
+              items:
+                $ref: '#/definitions/BuyerDashboardItem'
+            organisation:
+              type: string
+      BuyerDashboardItem:
+        type: object
+        properties:
+          applications:
+            type: integer
+          closed_at:
+            type: string
+          framework:
+            type: string
+          id:
+            type: integer
+          lot:
+            type: string
+          name:
+            type: string
+          status:
+            type: string
+    responses:
+      200:
+        description: Buyer dashboard data for the 'My briefs' tab
+        schema:
+          $ref: '#/definitions/BuyerDashboardItems'
+    """
+    organisation = users.get_user_organisation(current_user.email_address.split('@')[-1])
+    user_briefs = briefs.get_user_briefs(current_user.get_id())
+
+    return jsonify(items=user_briefs, organisation=organisation), 200
+
+
+@api.route('/buyers/team-briefs', methods=['GET'])
+@login_required
+@role_required('buyer')
+def get_buyer_dashboard_data_for_team_briefs():
+    """Buyer dashboard (Team briefs) (role=buyer)
+    ---
+    tags:
+      - dashboard
+    definitions:
+      BuyerDashboardTeamItems:
+        type: object
+        properties:
+            items:
+              type: array
+              items:
+                $ref: '#/definitions/BuyerDashboardTeamItem'
+            organisation:
+              type: string
+      BuyerDashboardTeamItem:
+        type: object
+        properties:
+          author:
+            type: string
+          closed_at:
+            type: string
+          framework:
+            type: string
+          id:
+            type: integer
+          lot:
+            type: string
+          name:
+            type: string
+          status:
+            type: string
+    responses:
+      200:
+        description: Buyer dashboard data for the 'Team briefs' tab
+        schema:
+          $ref: '#/definitions/BuyerDashboardTeamItems'
+    """
+    domain = current_user.email_address.split('@')[-1]
```

Comment:
rather than having to do this each time why not just do it in `get_user_organization`

Reply:
Domain also gets passed to `get_team_briefs` and `get_team_members`.